workflows/api/schema.graphql

```diff
@@ -7,7 +7,7 @@ input BoolComparators {
   _gte: Int
   _lt: Int
   _lte: Int
-  _is_null: Int
+  _is_null: Boolean
```

Comment:
@j-x-han @morsecodist is this right? This change showed up after I ran codegen in workflows/

Reply:
Yeah, I changed how the `is_null` comparator works back in this PR: https://github.com/chanzuckerberg/czid-platformics/pull/240